### PR TITLE
Put templates in js files

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: remoteappmanager/templates/home.html [hunk 1]
```diff
@@ -1,135 +1,11 @@
...
-  </form>
-  <!-- /.search form -->
-
-  <!-- Sidebar Menu -->
-  <ul id="applist" class="sidebar-menu">
-    <li class="header">APPLICATIONS</li>
-
-    <!--
-      We initialize this element with "display: none" to prevent displaying
-      "No applications found" and the loading spinner at the same time
```
Comment (martinRenou) [inline remoteappmanager/templates/home.html]: This has been removed

File: remoteappmanager/templates/home.html [hunk 2]
```diff
@@ -1,135 +1,11 @@
...
-    <!-- Loading spinner -->
-    <li v-show="model.loading" id="loading-spinner">
-      <a href="#">
-        <i class="fa fa-spinner fa-spin"></i>
-        <span>Loading</span>
-      </a>
-    </li>
-
-    <!-- Application list
-      The line 'v-show="true" style="display: none;"' is here to fix displaying
```
Comment (martinRenou) [inline remoteappmanager/templates/home.html]: idem

File: remoteappmanager/templates/home.html [hunk 3]
```diff
@@ -1,135 +1,11 @@
...
-  </ul>
-  <!-- /.sidebar-menu -->
-</section>
-<!-- /.sidebar -->
+  <div id="applist"></div>
 {% endblock %}
 
 {% block main %}
-  <!-- Application View
-      The line 'v-show="true" style="display: none;"' is here to fix displaying
```
Comment (martinRenou) [inline remoteappmanager/templates/home.html]: idem

File: remoteappmanager/templates/home.html [hunk 4]
```diff
@@ -1,135 +1,11 @@
...
-             :src="app.app_data.image.icon_128 | icon_src">
-
-        <button class="stop-button"
-                v-if="app.is_running()"
-                @click="model.stop_application(index)"
-                :disabled="app.is_stopping()">
-          <i class="fa fa-times"></i>
-        </button>
-
-        <span>{% raw %} {{ app.app_data.image | app_name }} {% endraw %}</span>
```
Comment (martinRenou) [inline remoteappmanager/templates/home.html]: `raw` helpers not used anymore

File: remoteappmanager/static/js/home/views/application_list_view.js [hunk 5]
```diff
@@ -3,10 +3,63 @@ define([
 ], function (Vue) {
     'use strict';
 
-    /* Create application_list ViewModel
-    (will next be wrapped in a main ViewModel which will contain the
-    applicationListView and the applicationView) */
-    var ApplicationListView = Vue.extend({});
+    var ApplicationListView = Vue.extend({
+        template:
```
Comment (stefanoborini) [inline remoteappmanager/static/js/home/views/application_list_view.js]: you can use the backticks `` for multiline strings.

Comment (martinRenou) [inline remoteappmanager/static/js/home/views/application_list_view.js]: That's what I did first, but it turned out that it's only available with ES6. So if you don't mind we'll do that later when using Webpack with browserify and babelify.

**Pull request merged**